merlin/models/tf/inputs/base.py

```diff
@@ -208,7 +200,13 @@ def InputBlock(
             name="continuous_projection",
         )
 
-    return ParallelBlock(branches, aggregation=aggregation, post=post, is_input=True, **kwargs)
+    _pre = PrepareFeatures(schema)
```

Comment:
does every model have PrepareFeatures as the first layer? if so, maybe we don't need this in the InputBlock?

Reply:
I changed that. As there are many blocks that don't use `InputBlockV2` and `InputBlock`, I moved the calling of `PrepareFeatures` to `Model`, `Encoder` and to the blocks that can be used as `pre` for `fit() / evaluate()` and the ones that can be used as a transform for `Loader.map()`